pkg/api/handlers/compat/networks.go

```diff
@@ -178,9 +178,7 @@ func CreateNetwork(w http.ResponseWriter, r *http.Request) {
 	network.Labels = networkCreate.Labels
 	network.Internal = networkCreate.Internal
 	network.IPv6Enabled = networkCreate.EnableIPv6
-
-	// FIXME use docker options and convert them to valid libpod options
-	// network.Options = networkCreate.Options
+	network.Options = networkCreate.Options
```

Comment:
This is dangerous.
First, we do not accept the actual docker options which makes it impossible to use the same compose file for docker and podman.
Second, the problem with using the raw options is that network create will error for invalid options. So potentially people are already using network options but we ignored them. Now it would error which can break there workflow.

I would prefer if we could translate the docker options to podman options.

Reply:
Ok. Generally speaking, how would I go about translating the docker options to podman options

Reply:
Loop over the [docker options](https://docs.docker.com/engine/reference/commandline/network_create/#bridge-driver-options) and check for them and set them to a proper podman network option.

`com.docker.network.bridge.mtu` can be mapped to mtu, `com.docker.network.bridge.name` should be set to network.InterfaceName.
`com.docker.network.bridge.enable_icc` is not yet implement (will be soon).

For the macvlan driver there are also some options but I could not find them documented clearly so we may have to look at the docker code to see what options are accepted there.

For the other and unknown options I think it would be best to ignore them but send a warning (see the warning field in the api response)

Reply:
Ok thank you, I'll give that a shot

Reply:
oh ok, I was looking at that file... thank you